src/main/java/seedu/duke/Duke.java

```diff
@@ -15,7 +16,11 @@ public static void main(String[] args) {
         Formatter.printWelcomeMsg();
 
         while (Ui.getIsRunning()) {
-            Formatter.printGoalBeforeShot(Ui.roundCount);
+            if (MatchStat.getIsMatchEnd()) {
+                Formatter.printAfterMatch();
```

Comment:
Formatter.printMatchResults() seems to be a better name, because the Formatter is in the if block where the statement already implies the match is over.